tests/js/sentry-test/fixtures/dashboard.js

```diff
@@ -2,9 +2,12 @@ import {Widget} from './widget';
 
 const DEFAULT_WIDGETS = [Widget()];
 
-export function Dashboard(widgets = DEFAULT_WIDGETS) {
+export function Dashboard(widgets = DEFAULT_WIDGETS, props = {}) {
   return {
-    widgets,
+    id: 1,
```

Comment:
Doesn't seem to affect the tests here but it may be good to make sure that `id` is a string rather than a number everywhere as it is declared.

Reply:
I can do that in the next pull request, as there are more tests coming.